tutor4points/core/static/css/home.css

```diff
@@ -0,0 +1,84 @@
+.dashboard-container {
+  display: flex;
+  border-style: solid;
+  border-width: 2px;;
+  border-color: #D6FF79;
+  border-radius: 2rem;
+  background-color: #323849;
+  padding: 50px;
+  margin: 3%;
+}
+
+.search-box {
+    display: flex;
+    padding: 50px; 
+    margin: 3%;
+}
+
+.search-field {
+    flex-grow: 3;
+}
+
+.search-button {
+    margin-left: 1em;
+}
+
+.list-container {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+}
+
+.info-container {
+    margin: 1em;
+    padding: 25px;
+    gap: 1em;
+    align-items: center;
+    border-style: solid;
+    border-color:#D6FF79;
+    border-width: 2px;
+    border-radius: 2rem;
+    background-color: #323849;
+}
+
+.profile-pic {
+    height:80px;
+    width:80px;
+    padding:5px;
+    border-radius: 50%;
+}
+.name-column{
+    font-size: 25px;
+    width:40%;
+}
+
+.classes-column{
+    font-size:20px;
+}
+
+.times-column{
+    font-size: 18px;
+    width:30%
+}
+
+.request-button{
```

Comment:
I suggest changing the request button color to purple so it matches the most updated button style of the view tutors page.